CMakeLists.txt

```diff
@@ -53,39 +53,30 @@ else (USB_LIBRARIES AND USB_INCLUDE_DIRS)
 
 endif (USB_LIBRARIES AND USB_INCLUDE_DIRS)
 
-
 find_package(catkin REQUIRED COMPONENTS
-  geometry_msgs
   tf
   roscpp
-  kdl_parser
-  exotica
-)
+  )
 
-find_package(Boost REQUIRED COMPONENTS system)
-find_package(orocos_kdl REQUIRED)
 catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES polhemus_ros
-  CATKIN_DEPENDS geometry_msgs tf roscpp kdl_parser exotica
-  DEPENDS system_lib orocos_kdl eigen
-)
-
+  LIBRARIES polhemus
```

Comment:
As above, this will block the polhemus name and potentially create conflict.

Reply:
It depends on whether you're also changing the name of the library. I would leave the name of the library as polhemus if it is only the liberty stuff.